app/models/middleware_server.rb

```diff
@@ -42,11 +42,11 @@ def in_domain?
 
   # Returns whether a server is immutable or not.
   #
-  # By default, we define all servers as being immutable, so that power
-  # operations are not allowed, and it is the provider responsability to
+  # By default, we define all servers as being mutable, so that power
```

Comment:
too much boolean logic here.
the default is an empty {}, which does not have the property and hence returns false which is _not_ immutable which is _mutable_

so the comment is right... once you meditate long enough 😄 

maybe drop or rephrase it in a later pr